statler/settings.py

```diff
@@ -108,3 +108,7 @@
 # https://docs.djangoproject.com/en/1.8/howto/static-files/
 
 STATIC_URL = '/static/'
+
+STATICFILES_DIRS = (
+    os.path.join(BASE_DIR, "static"),
+)
```

Comment:
I moved the static files from the folder `/statler_site/static` to `/static`. Making the folder location and the url the same was a simple way to make the image uploading work. This code tells Django about the static location.